src/docs/development/platform-integration/platform-channels.md

```diff
@@ -637,6 +636,52 @@ If using the iOS Simulator,
 note that it does not support battery APIs,
 and the app displays 'battery info unavailable'.
 
+## Typesafe platform channels via Pigeon {#pigeon}
```

Comment:
We should add a reference right at the top of the doc too listing that there are 2 choices. Otherwise it might be hard for people to find this.

Reply:
done